### DISCUSSION TITLE
error `An exception occurred: CLR/System.NullReferenceException` after fork sync

### DISCUSSION THREAD

**Discussion by onesch** [space-wizards/space-station-14#33677] [Questions] [answered]
Today I synchronized my fork with the local copy and continued working on my tasks, after which I saw that an error began to appear, I thought that I wrote something wrong, but no, in the branch where I conduct my “experiments”, this error also appears in the master branch ( I synchronized these two branches today), well, I think let’s check with the branch that was not synchronized today, it does not crash with this error, everything is ok in it.. why?

- when does the error appear?
I don’t do anything, I just stand there and after about a minute an error pops up.
```
An exception occurred: CLR/System.NullReferenceException
Unhandled exception of type 'System.NullReferenceException' in Content.Shared.dll: 'Object reference not set to an instance of an object.'
```

[Content.Shared\Random\Rules\NearbyTilesPercent.cs #43 line](https://github.com/space-wizards/space-station-14/blob/master/Content.Shared/Random/Rules/NearbyTilesPercent.cs#43)
![image](https://github.com/user-attachments/assets/76a19cac-8a16-4c95-a9d9-b5ebe2baf28b)

**Comment by Litraxx** [comment, marked as answer]
Seems to be getting fixed https://github.com/space-wizards/space-station-14/pull/33679
